packages/sdk/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## Recent Changes
 
+- Added unit tests for `SshConfigUtils`. [#614](https://github.com/zowe/zowe-native-proto/pull/614)
+- Resolved an issue where `~` was not being resolved to the home directory within ssh configuration files. [#614](https://github.com/zowe/zowe-native-proto/pull/614)
```

Comment:
Can we rewrite this so that we're not using "resolved" twice in the same sentence?